src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -35,6 +37,7 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ROLE + "STUDENT "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_COURSE + "CS2103T"
```

Comment:
Try running the command `add` without any argument (which is invalid). You'll see the error message being

```
Example: add n/John Doe p/98765432 e/johnd@example.com r/STUDENT a/311, Clementi Ave 2, #02-25 c/CS2103Tt/friends t/owesMoney
                                                                                                       ^^
```

There's an obvious missing space pointed out there.